slim.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.7-alpine as builder
+
+WORKDIR /girder
+RUN apk add --no-cache git gcc musl-dev libffi-dev openssl-dev
+COPY . ./
+RUN pip install --no-cache-dir .
+
+FROM python:3.7-alpine as runtime
+LABEL maintainer="Kitware, Inc. <kitware@kitware.com>"
+EXPOSE 8080
+
+# Set environment to support Unicode: http://click.pocoo.org/5/python3/#python-3-surrogate-handling
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+RUN apk add --no-cache tini
+
+COPY --from=builder /usr/local/bin/girder /usr/local/bin/girder
+COPY --from=builder /usr/local/lib/python3.7/site-packages/ /usr/local/lib/python3.7/site-packages/
+COPY --from=girder/girder:latest /usr/share/girder/static/ /usr/local/share/girder/static/
+
+ENTRYPOINT ["/sbin/tini", "-v", "--", "girder", "serve"]
```

Comment:
@brianhelba I don't think `tini` is strictly necessary in our case case because python as an entrypoint will already trap signals properly.  However it will collect zombies if girder creates them, and I don't think it hurts to include it here.

Without tini, it is my understanding that python itself would be pid 1, but girder starts with a python shebang, and that has to be interpreted by a shell, so maybe tini is needed after all....

Reply:
This isn't necessary. The [shebang is interpreted by the program loader itself](https://stackoverflow.com/a/3009280), not the shell.

I've verified via `docker exec girder ps` that `/usr/local/bin/python` is PID 1 if we execute `girder serve`.

Reply:
Verified that tini is necessary, and girder currently ignores sigterm https://github.com/girder/girder/issues/3247